js/image.js

```diff
@@ -0,0 +1,19 @@
+(function ($) {
+	var retries = 5;
+	$( document).ready(function(){
+		$('img').bind('error', function() {
+			var image = $(this);
```

Comment:
I'd prefix jQuery vars with a `$` so they are distinguishable from "normal" vars. We do this in our other code as well.